Added a background texture to ColorPicker

mrcdk opened the pull request:
So we can see how the alpha channel affects the color

Cleaned up version of [this PR](https://github.com/godotengine/godot/pull/11033) which I screwed before 😅 

It works on both the editor and the colorpicker gui control when added to a scene

mrcdk committed:
Added a background texture to ColorPicker

So we can see how the alpha channel affects the color

djrm commented on scene/resources/default_theme/source/icon_mini_checkerboard.svg:
```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```
this icon is currently in master, so no need to include it.

mrcdk replied:
It's the same icon for the scene default_theme

djrm replied:
oh, i see, you dont have to include it here then.

akien-mga commented:
There seems to be a merge conflict now.

mrcdk commented:
Ugh, what a pain, feel free to salvage anything this PR has I'm not touching this anymore

mrcdk commented:
I overreacted on my last comment, sorry. I've rebased the changes to master, resolved all merge conflicts and deleted the `icon_mini_checkerboard.svg` that was added to the scene's default theme source folder as requested by @djrm 

Now, can someone check that I didn't goofed anything else and merge ASAP so there can't be a chance for merge conflicts?

mrcdk commented:
Good think I asked to be merged ASAP before a merge conflict happen 😞 ... Both times by newer PR than this one (this one is almost 2 weeks old) well, I can't babysit this PR forever so I'm closing it.

akien-mga commented:
Geez, what's up with that attitude?

akien-mga commented:
Took me literally 4 seconds to fix the merge conflict. Now building locally (as you can see, Travis CI still hadn't built your PR and I don't merge stuff if I don't know it builds), and I'll do the merge myself since you don't care enough.

akien-mga commented:
Or no, I'll just wait for a contributor who cares enough about this not to give up when faced with a trivial merge conflict.
